src/renderer/components/Header.tsx

```diff
@@ -90,7 +90,7 @@ export const Header = memo(() => {
                         closeOnClick
                         closeOnMouseDown
                         borderRadius={8}
-                        label="Pause"
+                        label="Pause (F5)"
```

Comment:
I don't think "Pause" also using F5 is a good idea. Users might accidentally double tab the button. It might also be that chainner takes a while to start running a chain, so users might get impatient and press the button again.